### PR TITLE
update(SuggestedEmoji): Improve Suggested Emoji UI

### LINKED ISSUE #1545: chore(emoji suggestions): Change emoji suggestions design
Make the design of the emoji suggestions more consistent with the reactions UI. 

Reactions:
Image

Current Emoji Suggestion UI:
Image

Emoji Suggestion Mockup:
Image

Figma Link: https://www.figma.com/file/GrAhLmQWA2kfD1Nw9nd24m/Chats-%2F-Landing-Page?type=design&node-id=1482-40987&mode=design&t=wd0EGKsne0UrZv0w-4

If you have any questions,, please don't hesitate to ask.

### PR DESCRIPTION
**Pull request by lgmarchi**
### 1. Improve UI in Suggested Emoji 
image

### Which issue(s) this PR fixes 🔨

- Resolve #1545

### CODE REVIEW

**Review by sdwoodbury — APPROVED** [review]

**Review by dariusc93 — APPROVED** [review]

File: ui/src/layouts/chats/style.scss [hunk 1]
```diff
@@ -759,11 +760,18 @@
...
     &:hover {
       cursor: pointer;
-      background-color: var(--secondary-dark); 
+      background-color: var(--secondary-light); 
     }
   }
+
+  .btn-wrap {
+    position: absolute;
+    top: var(--gap);
```
Comment (InfamousVague) [inline ui/src/layouts/chats/style.scss:778]: I think we will want `gap-less` here

File: ui/src/layouts/chats/style.scss [hunk 2]
```diff
@@ -759,11 +760,18 @@
...
+    right: var(--gap);
```
Comment (InfamousVague) [inline ui/src/layouts/chats/style.scss:779]: same here

**Review by InfamousVague — CHANGES_REQUESTED** [review]
Some small CSS things might need work, I can help tidy them up a bit if needed

**Comment by phillsatellite** [comment]
Noticed a few small issues 

- It looks like the emoji selection box doesn't come out as far as the mockup UI does 

Screenshot 2023-11-27 at 1 42 05 PM

Screenshot 2023-11-27 at 1 42 13 PM

(emoji selection box) 
Screenshot 2023-11-27 at 1 45 15 PM

(reply modal)
Screenshot 2023-11-27 at 1 45 56 PM

Also noticed the X button was a little off and intruded into the highlight bar (@XileHorizon is working to see if he wants to modify the design a little for this part)
Screenshot_2023-11-27_at_1 25 03_PM

**Comment by InfamousVague** [comment]
image

**Review by InfamousVague — APPROVED** [review]

**Review by phillsatellite — APPROVED** [review]

**Review by stavares843 — APPROVED** [review]

**Pull request merged**